res/games/gameitem.jinja

```diff
@@ -0,0 +1,120 @@
+{%- macro playermacro(player, me, iconpath, teamalign) %}
+        {% set player_login = player.login %}
+        {% set player_country = "" %}
+        {% set player_global_rating = "???" %}
+        {% if player.login is defined %}
+            {% set player_country = iconpath~player.country~".png" %}
+            {% set player_global_rating = player.rating_estimate %}
+        {% endif %}
+
+        {% set width = 0 %}
+            {% if "-" in player.login %}
+                {% set width = 200 %}
+            {% endif %}
+
+        {% if player_login is not defined %}
+            {% set player_login = player %}
+        {% endif %}
+
+        {% if me.login is defined and player.login == me.login %}
+            {% set player_login = "<b><i>"~player_login~"</b></i>" %}
+
+        {% elif player.clan == me.clan and player.clan is not none %}
```

Comment:
Missed a 'me' check compared to previous one. Anyway, do jinja docs say something about comparing with undefined values? If they don't compare with anything, we might get away with not checking 'me' here.

Reply:
wym missing? there is a check checking me.login that accounts for all possible anomalies that could happen to me, right?

Reply:
wym missing? there is a check checking me.login that accounts for all possible anomalies that could happen to me, right?

Reply:
If 'me' is undefined, then you execute the 'elif', so you can end up using undefined 'me' there. Anyway jinja code suggests that undefined values compare falsely with defined values, so as long as we compare 'me' stuff with defined values we should be fine.
As in, compare with player_login in that first loop, and check if player.clan is defined before the comparison.

Reply:
Well, clan is set to None in the player.py, so it is always defined, isnt it? i dont really get what i should change about that now, you mean i should put both the login==me and the clan==me check into a bigger me=defined check instead of having the clan==me check as an elif?